Formula/code-server.rb

```diff
@@ -0,0 +1,22 @@
+class CodeServer < Formula
+  desc "Access VS Code through the browser"
+  homepage "https://github.com/cdr/code-server"
+  url "https://registry.npmjs.org/code-server/-/code-server-3.3.0-rc.7.tgz"
+  sha256 "22b7df81f2a5d372bac5bf5251407ca18493675ee3d50113b4d7817b711503b2"
+
+  depends_on "python" => :build
```

Comment:
Sorry if miss it, but can we use python 3.8 here?

```suggestion
  depends_on "python@3.8" => :build
```

Reply:
Is there a reason we need to version it? The brew formula guide for node packages uses python without a version.

Reply:
We're gradually migrating all the formulae to [`python@3.8`](https://github.com/Homebrew/homebrew-core/issues/47274) (default `python` formula is 3.7). When we're done with migrating we'll switch `python` to 3.8 (that's why docs are still pointing to `python`).
In the meantime, we ask to use python 3.8 where's possible

Reply:
Ok you got it.